src/views/LobbyView.vue

```diff
@@ -94,12 +99,10 @@ export default {
       return this.gameStore.name;
     },
     iAmReady() {
-      return this.gameStore.myPNum === 0
-        ? this.gameStore.p0Ready
-        : this.gameStore.p1Ready;
+      return this.gameStore.myPNum === 0 ? this.gameStore.p0Ready : this.gameStore.p1Ready;
     },
     readyButtonText() {
-      return this.iAmReady ? 'UNREADY' : 'READY';
+      return this.iAmReady ? this.t('lobby.unready') : this.t('lobby.ready');
```

Comment:
Issue: avoid duplication
```suggestion
      return this.t(this.iAmReady ? 'lobby.unready' : 'lobby.ready') 
```